packages/fast-components-react-msft/src/typography/index.ts

```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+import {
```

Comment:
why do we need all these interface imports? we don't seem to be re-exporting them etc

Reply:
Good question - this is the pattern in every other component.

Reply:
I'm pretty sure Typescript makes us.